app/assets/stylesheets/characters.scss

```diff
@@ -7,6 +7,10 @@
 	padding-top: 20px;
 }
 
+.characters-section.characters-padding.characters-top-bottom {
```

Comment:
I don't feel like `characters-padding` is a good class name, it is too attached  to the css property